### PR TITLE
(fixes #210) Fixing error for not installed packages

### LINKED ISSUE #210: Error message for not-installed package isn't clear
Currently if you don't have the package installed that you're trying to run pkgnet on, the error messages look something like:
```
INFO [2019-03-28 19:02:47] Checking installed packages...
FATAL [2019-03-28 19:02:47] pkgnet could not find a package called 'data.table'.
```
We should have the FATAL error message give the instruction that the user should install the package first.

### PR DESCRIPTION
**Pull request by shivamx96**

### CODE REVIEW

**Comment by shivamx96** [comment]
Changed error message when pkgnet is run on a package that is not-installed on the system.
[](https://github.com/UptakeOpenSource/pkgnet/issues/210)

File: R/AbstractPackageReporter.R [hunk 1]
```diff
@@ -171,8 +171,8 @@ AbstractPackageReporter <- R6::R6Class(
             )
 
             if (! pkg_name %in% installed_packages){
-                msg <- sprintf("pkgnet could not find a package called '%s'.", pkg_name)
-                log_fatal(msg)
+              msg <- sprintf("pkgnet could not find a installed package named '%s'. Please install the package first.", pkg_name)
```
Comment (jameslamb) [inline R/AbstractPackageReporter.R]: thanks! can you please change `a installed` to `an installed`

**Comment by jameslamb** [comment]
Thanks @shivamx96 ! Would you please add `(fixes #210)` to the title of this pull request? If you do that, then GitHub will automatically add a link to this PR on issue #210 !

We like to do that to keep a record of which PR(s) addressed particular issues. As an added bonus, if you do that GitHub will also automatically close the issue once we merge your PR.

File: R/AbstractPackageReporter.R (hunk 1 above)
Comment (shivamx96) [reply R/AbstractPackageReporter.R]: Sure, will do.

**Comment by jameslamb** [comment]
I apologize @shivamx96 , I misled you on the issue. It appears we _do_ have unit tests checking this error message. Can you please change them?

To see which tests are failing, run `devtools::test()` in R with your working directory set to the root of this repo.

File: R/AbstractPackageReporter.R (hunk 1 above)
Comment (shivamx96) [reply R/AbstractPackageReporter.R]: Changed that as well as the error message check in `tests/testthat/test-AbstractPackageReporter.R`. Please have a look.

**Comment by jayqi** [comment]
Looks good, thanks for the PR and making the requested changes @shivamx96.

**Pull request merged**